Mathlib/RingTheory/Polynomial/DegreeLT.lean

```diff
@@ -0,0 +1,238 @@
+/-
+Copyright (c) 2025 Kenny Lau. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Anne Baanen, Kenny Lau
+-/
+
+import Mathlib.LinearAlgebra.Determinant
+import Mathlib.LinearAlgebra.Matrix.Block
+import Mathlib.RingTheory.Polynomial.Basic
+
+/-!
+# Polynomials with degree less than `n`
+
+This file contains the properties of the submodule of polynomials of degree less than `n` in a
+commutative ring `R`, denoted `R[x]_n`.
+
+## Main definitions/lemmas
+
+* `degreeLT.basis R n`: a basis for `R[x]_n` the submodule of polynomials with degree `< n`,
+given by the monomials `X^i` for `i < n`.
+
+* `degreeLT.basis_prod R m n`: a basis for `(R[x]_m) × (R[x]_n)` as above, which is the sum
+of the two bases.
+
+* `degreeLT.addLinearEquiv R m n`: an isomorphism between `(R[x]_(n+m))` and `(R[x]_n) × (R[x]_m)`
+given by the fact that the bases are both indexed by `Fin (n+m)`.
+
+* `translate a`: (To move?) An automorphism of `R[X]` given by a substitution of the form `X ↦ X - a`.
+
+* `translateLinear a n`: The linear automorphism induced by `translate` on `R[x]_n`.
+
+-/
+
+
+namespace Polynomial
+
+/-- R[x]_n is notation for the submodule of polynomials of degree strictly less than n. -/
+scoped notation:9000 R "[x]_" n => Polynomial.degreeLT R n
```

Comment:
This should surely be a capital X in the notation, R[X] is the notation for polynomials.